README.md

```diff
@@ -543,4 +554,11 @@ to
 ```YAML
  uses: dvsa/.github/.github/workflows/php-library-tests.yml@v3.2.3
 
-```
\ No newline at end of file
+```
+
+Secrets:
+* [REQUIRED] `SNYK_TOKEN` - one can be inherited from the organisation - contact shaun.hare@dvsa.gov.uk or a DVSA member of your team to get that enabled.
+
+Inputs:
+* [REQUIRED] `php-version` - list of strings of all php versions which the workflow will run against
+* [OPTIONAL] `fail-fast` - boolean to control whether all steps in the matrix should be run, even if one of them fails
```

Comment:
missing php library only

Reply:
.github/workflows/php-tests.yml also uses matrix strategy and I've added `fail-fast` to both.